app/controllers/application_controller.rb

```diff
@@ -5,6 +5,67 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions unless test?
+    set :session_secret, "secret"
   end
 
+  get '/' do
+  erb :index
+  end
+
+  get '/tweets/new' do
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller).